Frameworks/CoreFoundation/PlugIn.subproj/CFBundle_Locale.c

```diff
@@ -675,9 +675,10 @@ static CFStringRef _CFBundleCopyLanguageFoundInLocalizations(CFArrayRef localiza
 
 // WINOBJC: Helper functions for workaround in _CFBundleCreateMutableArrayOfFallbackLanguages
 static CFStringRef _copyStringTruncated(CFStringRef localization, CFRange cutoff) {
-    CFMutableStringRef truncatedString = CFStringCreateMutableCopy(NULL, 0, localization);
-    CFStringDelete(truncatedString, CFRangeMake(cutoff.location, CFStringGetLength(truncatedString) - cutoff.location));
-    return truncatedString;
+    return CFStringCreateWithSubstring(NULL, localization, CFRangeMake(0, cutoff.location));
+    //CFMutableStringRef truncatedString = CFStringCreateMutableCopy(NULL, 0, localization);
```

Comment:
commented!

Reply:
Oh woops. I wanted to keep that around in case I goofed anything in the substring call.